ddtrace/contrib/asyncpg/patch.py

```diff
@@ -43,6 +43,13 @@
 
 log = get_logger(__name__)
 
+def get_version():
+    import asyncpg
```

Comment:
Do we need to inline this import?

Reply:
I'd recommend removing this and making the import on line 29 unconditional.